File: remove-element.py
class Solution(object):
    def removeElement(self, nums, val):
        """
        :type nums: List[int]
        :type val: int
        :rtype: int
        """
        k=[]
        for i in range(len(nums)):
            if nums[i]==val:
                k.append(i)
        for i in reversed(k):
            nums.pop(i)

File: remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int poi = 0;
        for(int i = 0; i< nums.length; i++){
            if(nums[i]!= val){
                nums[poi] = nums[i];
                poi++;
            }
        }return poi;
    }
}